### PR TITLE
Replace SCROLLABLE_ROW_HEIGHT define with constexpr kScrollableRowHeight

### PR DESCRIPTION
**Pull request by katerid**
Replacing SCROLLABLE_ROW_HEIGHT with constexpr and kScrollableRowHeight

Part of #21421

### CODE REVIEW

**Comment by katerid** [comment]
I believe that this should be ready for review, the only issue is that one of my commits had too short of a name.

File: src/openrct2/network/NetworkBase.cpp [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Gymnasiast) [inline src/openrct2/network/NetworkBase.cpp:1]: Unrelated change, please revert.

File: src/openrct2/command_line/RootCommands.cpp [hunk 2]
```diff
@@ -487,4 +487,4 @@ static void PrintLaunchInformation()
     Console::WriteLine();
 
     // TODO Print other potential information (e.g. user, hardware)
-}
+}
```
Comment (Gymnasiast) [inline src/openrct2/command_line/RootCommands.cpp:490]: Unrelated change, please revert.

File: src/openrct2/Version.h [hunk 3]
```diff
@@ -84,4 +84,4 @@ struct NewVersionInfo
     std::string url;
 };
 
-NewVersionInfo GetLatestVersion();
+NewVersionInfo GetLatestVersion();
```
Comment (Gymnasiast) [inline src/openrct2/Version.h:87]: Unrelated change, please revert.

File: .vscode/c_cpp_properties.json [hunk 4]
```diff
@@ -1,85 +0,0 @@
-{
```
Comment (Gymnasiast) [inline .vscode/c_cpp_properties.json:1]: Unrelated change, please revert.

**Review by Gymnasiast — CHANGES_REQUESTED** [review]
There are a number of changes in this PR that shouldn’t be there, like one deleted file, two instances where the trailing newline was removed, and one case of an extra newline. Please fix these. The actual meat of the PR does look good, though.

**Comment by katerid** [comment]
> There are a number of changes in this PR that shouldn’t be there, like one deleted file, two instances where the trailing newline was removed, and one case of an extra newline. Please fix these. The actual meat of the PR does look good, though.
> 
> Also, please do not close PRs only to recreate them - just update your existing one.

Sorry about the closed PR, I was just changing which file I was working on and I'm still newer to github. I didn't know how to take out some of my old commits and only include the new ones that were relevant to this PR which was my goal with creating a new one.

**Comment by Gymnasiast** [comment]
No worries, we have all had to learn it.

Removing unwanted commits can be done using a rebase. It is an advanced operation though.

A tip for future PRs: make you make your changes on a branch, rather than the `master` branch. This keeps things cleaner and avoids unrelated commits stacking up.

**Comment by Gymnasiast** [comment]
@katerid Could you address the line notes?

**Comment by AaronVanGeffen** [comment]
I intended to rebase this PR, but it looks like the `SCROLLABLE_ROW_HEIGHT` define has already been replaced with a constexpr `kScrollableRowHeight` in #21760. Therefore closing this PR.

**Pull request closed without merging**